From 9f5b9dbb654a8401c6d0f49715e3839f1e20c8d3 Mon Sep 17 00:00:00 2001
From: Pitchaiah P <pitchaiah.p@variscite.com>
Date: Sun, 13 Jul 2025 22:39:06 +0200
Subject: [PATCH] These changes assume that the product name of the NXP iMX8MN
 derivatives include the string "8mn" and  this is used to simplify the checks
 in the build script.

Derived from
https://github.com/varigit-dev/MX6x-android/blob/android-14.0.0_1.0.0-var01/scripts/platform/device/nxp.git/0002-android-imx_device_fsl-allow-building-any-NXP-iMX8MN.patch

Signed-off-by: Pitchaiah P <pitchaiah.p@variscite.com>
---
 common/tools/imx-make.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/common/tools/imx-make.sh b/common/tools/imx-make.sh
index 910b4eda..0e6b1b75 100755
--- a/common/tools/imx-make.sh
+++ b/common/tools/imx-make.sh
@@ -198,7 +198,7 @@ if [ ${build_android_flag} -eq 1 ] || [ ${build_whole_android_flag} -eq 1 ]; the
     fi
     TARGET_IMX_KERNEL=true make ${parallel_option} ${build_bootimage} ${build_vendorbootimage} ${build_dtboimage} ${build_vendordlkmimage} || exit
     if [ -n "${build_bootimage}" ] || [ ${build_whole_android_flag} -eq 1 ]; then
-        if [ ${TARGET_PRODUCT} = "evk_8mp" ] || [ ${TARGET_PRODUCT} = "evk_8mn" ] \
+        if [ ${TARGET_PRODUCT} = "evk_8mp" ] || [ ${TARGET_PRODUCT} =~ "8mn"  ] \
         || [ ${TARGET_PRODUCT} = "evk_8ulp" ] || [ ${TARGET_PRODUCT} = "mek_8q" ] \
         || [ ${TARGET_PRODUCT} = "evk_8mm" ] || [ ${TARGET_PRODUCT} = "evk_8mq" ] \
         || [ ${TARGET_PRODUCT} = "evk_95" ]; then
-- 
2.43.0

